docs/examples/dcgan.md

```diff
@@ -0,0 +1,148 @@
+Deep Convolutional Generative Adversarial Networks are a class of CNN and one of the first approach which made GAN "stable" and usable to learn features from images in unsupervised learning. This project is a porting from the [pytorch/examples/dcgan](https://github.com/pytorch/examples/tree/master/dcgan). At the end of this example you will be able to use DCGAN for generating images from your dataset.
+
+![Generated images](../img/lfw-300epochs.gif)
+
+In this guide we will train a [Pytorch](http://pytorch.org/) model in an unsupervised task and use it to
+generate images from an input vector Z(100 dimensional uniform distribution). Finally we will deploy the trained model as a
+REST endpoint that can be used to generate images in real time.
+
+## Project setup
+
+The code for this project is available on this [Github repository](https://github.com/ReDeiPirati/dcgan.git). Once [logged in](../commands/login), clone the project and [initialize](../commands/init) a floyd project.
+
+```bash
+$ git clone https://github.com/ReDeiPirati/dcgan.git
+$ cd dcgan
+$ floyd init dcgan
+```
+
+## Training
+
+### Dataset
+
+For this project we will use the [Labeled Faces in the Wild Home](http://vis-www.cs.umass.edu/lfw/) aka LFW for the training.
+Since we are learning in an unsupervised regime, there is not a train/dev/test split, but we use the BCE Loss Error of the Discriminator and Generator as good metrics to learn(minmax game strategy from Game Theory), moreover we can visualize the generated images every epoch.
+This preprocessed dataset is available publicly on
+[FloydHub](https://www.floydhub.com/search/datasets?query=lfw).
+
+
+### Training
+
+You can train the DCGAN model by running `main.py` script with required
+parameters. Below is the [command](../commands/run) to start a training job on Floyd:
+
+```bash
+$ floyd run --gpu --env pytorch --data <USER>/datasets/lfw/<VERSION>:lfw "python main.py --dataset lfw --dataroot /lfw --outf /output --cuda --ngpu 1 --niter 20"
```

Comment:
Nitpick, I think it's fine to hardcode and promote your own dataset name here :)

This makes it easier for users to try it out too since they can skip the search process and jump right into the training.

Reply:
I totally agree with you, this sounds more KISS like!